src/renderers/webxr/WebXRManager.js

```diff
@@ -188,6 +193,24 @@ class WebXRManager extends EventDispatcher {
 
 		};
 
+		this.getBaseLayer = function () {
+
+			return glProjLayer !== null ? glProjLayer : glBaseLayer;
+
+		};
+
+		this.getBinding = function () {
+
+			return glBinding;
+
+		};
+
+		this.getxrFrame = function () {
```

Comment:
Isn't `frame` available already in the `onAnimationFrame()` callback?

Reply:
Yes, but then it has to be passed into the components that would like to have access.
For instance, I was experimenting with creating a cube layer for the background and it was handy to have it available on the `xr` object.

Reply:
@mrdoob I can remove it if you prefer

Reply:
I see...

How following the naming pattern of the other methods and call it `getFrame()`?

Reply:
done